clip_benchmark/cli.py

```diff
@@ -50,9 +52,13 @@ def run(args):
     if args.skip_load:
         model, transform, collate_fn, dataloader = None, None, None, None
     else:
-        model, _, transform = open_clip.create_model_and_transforms(args.model, pretrained=args.pretrained, cache_dir=args.model_cache_dir)
-        model = model.to(args.device)
-        tokenizer = open_clip.get_tokenizer(args.model)
+        if args.ja_clip:
```

Comment:
could you put both these branches in a load_clip function taking as input 

* model_type that takes for value either "ja_clip", either "open_clip"
* pretrained
* device
* model_cache_dir
* model

that way it should be possible to expand this to more clip variants without making this part of the code too long

Reply:
thanks for your feedback. how about this? 
https://github.com/LAION-AI/CLIP_benchmark/pull/50/commits/764aa54fd13a47733d860dc4e39d599d28c644e8